utils/spiel/m0spiel

```diff
@@ -700,7 +700,7 @@ The following example illustrates usage of the tool:
 
 Execute commands from command line:
 
-\techo 'print(spiel_service_init(Fid(1,1)))' | sudo ./m0spiel''')
+\techo 'print(service_init(Fid(1,1)))' | sudo ./m0spiel''')
```

Comment:
Please add some empty lines after this line.
This line and above lines are help messages.
And the next line is start argument parsing.
It's better to have some empty lines to help reading.

Reply:
Done, thanks.